src/test-helpers/click.ts

```diff
@@ -1,11 +1,13 @@
-import { act } from '@testing-library/react'
-import userEvent from '@testing-library/user-event'
+import { act, fireEvent } from '@testing-library/react'
 
-/** Clicks an element within an act block. */
+/** Clicks an element within an act block, simulating mousedown and mouseup separately. */
 const click = async (selector: string) => {
   const el = document.querySelector(selector)!
   await act(async () => {
-    userEvent.click(el)
+    fireEvent.mouseDown(el)
+  })
+  await act(async () => {
+    fireEvent.mouseUp(el)
   })
```

Comment:
Does this work in a single `act`? That would be more performant.

Reply:
I did test that, but it only works as expected in a separate act.